tests/test_components/test_autograd.py

```diff
@@ -703,6 +703,35 @@ def objective(*args):
         ag.grad(objective)(params0)
 
 
+@pytest.mark.parametrize("colocate", [True, False])
+@pytest.mark.parametrize("objtype", ["flux", "intensity"])
+def test_interp_objectives(use_emulated_run, colocate, objtype):
+    monitor = td.FieldMonitor(
+        center=(0, 0, 0),
+        size=(td.inf, td.inf, 0),
+        freqs=[FREQ0],
+        name="monitor",
+        colocate=colocate,
+    )
+
+    def objective(args):
+        structures_traced_dict = make_structures(args)
+        structures = list(SIM_BASE.structures)
+        for structure_key in structure_keys_:
+            structures.append(structures_traced_dict[structure_key])
+
+        sim = SIM_BASE.updated_copy(monitors=[monitor], structures=structures)
+        data = run(sim, task_name="autograd_test", verbose=True)
+
+        if objtype == "flux":
+            return anp.sum(data[monitor.name].flux.values)
+        elif objtype == "intensity":
+            return anp.sum(data.get_intensity(monitor.name).values)
+
```

Comment:
in view of the notebook failing, maybe we need to add a `parameterize` for the size of the monitor (planar vs point?)

Reply:
Will do. The test doesn't fail for point monitors though 😂